styleguide/Repositories/PromoRepository.php

```diff
@@ -53,6 +53,14 @@ public function getRequestData(array $data)
         // Only pull accordion for childpage template
         $accordion = !empty($accordion_page_ids[$data['page']['id']]) ? app('Factories\Accordion')->create($accordion_page_ids[$data['page']['id']]) : null;
 
+        $grid_page_ids = [
```

Comment:
In reference to the comment above about having to select the `GridController` anyway, all of the changes to this file shouldn't be necessary since they are handled by the `GridController` and `GridRepository`. The styleguide page you added `styleguide/Pages/Grid.php` already points to the `GridController` so that should be good to go!